Source/ResponseSerialization.swift

```diff
@@ -189,7 +189,7 @@ extension DataRequest {
         appendResponseSerializer {
             // Start work that should be on the serialization queue.
             let start = CFAbsoluteTimeGetCurrent()
-            let result = AFResult { try responseSerializer.serialize(request: self.request,
+            let result = Result<Serializer.SerializedObject, Error> { try responseSerializer.serialize(request: self.request,
```

Comment:
Please adjust the parameter alignment, and perhaps put the `try` on its own line to it isn't as wide.

Reply:
Done.